product_docs/docs/biganimal/release/using_cluster/pgd_cli_ba.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: PGD CLI on BigAnimal
+navTitle: PGD CLI on BigAnimal
+deepToC: true
+---
+
+When running a distributed high-availability cluster on BigAnimal, you can use the [PGD CLI](../../../pgd/latest/cli/) to manage cluster operations, such as switching over write leaders, performing cluster health checks, and viewing various details about nodes, groups, or other aspects of the cluster.
+
+Using the PGD CLI with BigAnimal requires installing it on a client machine, so [install the CLI](../../../pgd/latest/cli/installing_cli/) if you haven't already.
```

Comment:
Also why aren't you giving them install instructions? At least at a cursory level.